t/basic.t

```diff
@@ -4,6 +4,23 @@ use Test::More;
 use Test::Mojo;
 
 my $t = Test::Mojo->new('Qi');
-$t->get_ok('/')->status_is(200)->content_like(qr/Mojolicious/i);
+$t->get_ok('/')->status_is(200);
+
+$t = $t->post_ok('/api/assets' => {Accept =>'application/json' , dataType => 'json'} => json => {tag => '000000A' , add_stamp => '2016-08-10 16:21:21'})
+    ->status_is(200);
```

Comment:
Tried for a long time to test for values, but could never seem to get the tests to pass. Definitely wanted to have that work as well, but I can't seem to nail it down.